<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCnaF">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Celsjusze na Fahreheity &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFnaC">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt; Fahrenheity na Celsjusze</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinFahr">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Celsjusze</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fahrenheity</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionZako_cz"/>
   </widget>
   <widget class="QMenu" name="menuKonwertowanie">
    <property name="title">
     <string>Konwertowanie</string>
    </property>
    <addaction name="actionC_na_F"/>
    <addaction name="actionF_na_C"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuKonwertowanie"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionC_na_F">
   <property name="text">
    <string>C na F</string>
   </property>
  </action>
  <action name="actionF_na_C">
   <property name="text">
    <string>F na C</string>
   </property>
  </action>
  <action name="actionZako_cz">
   <property name="text">
    <string>Zako≈Ñcz</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
